### PR TITLE
🐛 Require image-spec v1.0.2 

### PR DESCRIPTION
**Pull request by kashifest**
Address security  vulnerability GHSA-77vh-xpmg-72qh

### CODE REVIEW

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -33,3 +33,5 @@ require (
 replace github.com/metal3-io/cluster-api-provider-metal3/api => ./api
 
 replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.4.4
+
+require github.com/opencontainers/image-spec v1.0.2 // indirect
```
Comment (furkatgofurov7) [inline go.mod:37]: qq: Why not to add it in the **require** list?

Comment (kashifest) [reply go.mod:37]: Wanted to make it separate so that its easy to remove in future

**Pull request merged**